Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```diff
@@ -202,6 +213,17 @@
   - type: Thieving
     stripTimeReduction: 1
     stealthy: true
+  - type: NinjaGloves
+  - type: NinjaDoorjack
+  - type: NinjaDrain
+  - type: NinjaStun
+  - type: NinjaDownload
+  - type: NinjaTerror
+  # not actually electrified, just used to make stun ability work
+  - type: Electrified
```

Comment:
this sounds like a hack

Reply:
well you cant collide with gloves so it cant troll...